src/utils/write-operations.ts

```diff
@@ -59,13 +59,14 @@ export const writeOperationsRepository = (
     const airseekerBasePath = join(api3BasePath, 'airseeker');
     mkdirSync(airseekerBasePath, { recursive: true });
 
-    Object.entries(payload.api3.airseeker).forEach(([filename, airseekerConfigs]) => {
-      const airseekerConfigBasePath = join(airseekerBasePath, filename);
-      mkdirSync(airseekerConfigBasePath);
-      Object.entries(airseekerConfigs).forEach(([filename, airseekerConfig]) => {
-        writeJsonFile(join(airseekerConfigBasePath, filename), airseekerConfig);
-      });
-    });
+    // TODO enable once airseeker is deployed
```

Comment:
I've addressed this [here ](https://github.com/api3dao/operations/pull/26/files#diff-7dbe290607c575e152c364310b0b12b9bb9b574915ce6e0c6c23fc73fe156cc2R61). So I don't think you need to do this anymore

Reply:
All good thanks :)